editors/vile/Portfile

```diff
@@ -20,12 +20,14 @@ long_description    vile is a text editor which is extremely compatible \
                     interpreter, and robust support for non-Unix hosts.
 
 homepage            https://invisible-island.net/vile/
-master_sites        ftp://ftp.invisible-island.net/pub/${name}/ \
-                    https://invisible-mirror.net/archives/${name}/
+master_sites        ftp://ftp.invisible-island.net/pub/${name}/current/ \
+                    https://invisible-mirror.net/archives/${name}/current/
 extract.suffix      .tgz
 
-checksums           sha1    d99f41479fa9726147301193a0e10e4282e4779c \
-                    rmd160  83a0a76be2e2914a193de9c61b7f1a7d159068df
+checksums           sha1    1c102d1bc6b3c0077ecb35afd0772d5c75502b5f \
```

Comment:
Please remove sha1 from the checksums, we don’t require it any more.

Reply:
…and done.